code-challenges.test.js

```diff
@@ -50,3 +102,21 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+
+// Pseudo Code: I really struggled with this problem and needed assistance with the whole thing. I needed a lot of guidance on how to make the .filter method work for me and I want to go to office hours this week so that I can brush up on my javascript because I struggled with the javascript portion of my assessment the most. 
+
+// Process: created a method that takes in a hand and uses the .filter method to check if the card matches the value of the specified index.
+// Variables: I created two variables defining how many numbers a pair and a three of a kind would have using the .includes method.
+// Return: I told it to return and show me if the hand met the specified parameter of needing a pair and a 3 of a kind to be displayed.
+
+const handChecker = (hand) => {
+    let checker = hand.filter (card => card === hand[0])
+    let pair = checker.includes(2)
+    let threeOfAKind= checker.includes(3)
+    return pair &&threeOfAKind
```

Comment:
Here you are using .filter on the inputted array, which is returning a new array with only the cards that match what the card at index 0 was in the original array. Let's say the array is [5, 6, 6, 5, 6]. Line 113 will return [5, 5], since index 0 of the original array is 5. Line 114 checks if this new array includes the number 2, which it does not, so it will return false. Line 115 will check if it includes the number 3, which is does not. Instead of seeing if it includes a specific number, we can check for the length of the new array. If the new array's length is 2, we know we have a pair. We then need to figure out a way to check if the rest of the numbers in the original array are a three of a kind. I think that is a good starting point for you, I have full confidence you are able to figure this out!